src/collective/fingerpointing/upgrades/configure.zcml

```diff
@@ -0,0 +1,29 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:gs="http://namespaces.zope.org/genericsetup">
+
+  <include file="profiles.zcml" />
+
+  <gs:registerProfile
```

Comment:
i'd like to learn more about testing upgrade steps:
you'll have to test the complete upgrade path from all generic setup profile numbers that ever have been used (eg. 1,2,3,4,10,21) to the most recent one: 1 to 21, 2 to 21, 3 to 21, 4 to 21 and 10 to 21
how do you "prepare" the database for running the upgrade so it reflects exactly the configuration that is present if this the addon has been installed with that GS profile version?


by registering the profile for `Products.CMFPlone.interfaces.IMigratingPloneSiteRoot` you automatically hide the profile in quickinstaller and it only shows up in portal_setup's upgrades tab